core/src/main/java/org/infinispan/container/impl/KeyValueMetadataSizeCalculator.java

```diff
@@ -17,5 +18,18 @@
     * @param metadata The metadata for this entry to be used in size calculation
     * @return The size approximately in memory the key, value and metadata use.
     */
-   long calculateSize(K key, V value, Metadata metadata);
+   default long calculateSize(K key, V value, Metadata metadata) {
```

Comment:
This is not a public method, so there's no reason to keep the old signature around